src/blackmetal/src/communication/controlSW/Odometry.hpp

```diff
@@ -0,0 +1,147 @@
+#pragma once
+
+#include <memory>
+#include <thread>
+class Odometry;
```

Comment:
Preco tato deklaracia?

Reply:
Musi to tam byt lebo tam je cyklicka dependencia. Bez tejto deklaracie by to neslo skompilovat.

Reply:
som sa trocha pohras includmi a uz to tam nemusi byt